Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h

```diff
@@ -510,6 +518,9 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
   double LineThickness;
   double LineDiameter;
 
+  int VaryRadius;
+  double RadiusFactor;
```

Comment:
This RadiusFactor should not be necessary for several reasons:
- If we need some kind of relative sizing then LineDiameter should be sufficient.
- If some non-trivial mapping is needed then I don't think factors are sufficient, but we would need an offset as well. This could be added later, if there is a strong driving use case.
- vtkTubeBender only supports a single scaling factor, so we need to compute a new array from the raw input array anyway.

Reply:
Agree, fixed